presto-main/src/main/java/com/facebook/presto/server/CoordinatorModule.java

```diff
@@ -85,6 +85,7 @@
 import com.facebook.presto.metadata.CatalogManager;
 import com.facebook.presto.operator.ForScheduler;
 import com.facebook.presto.server.protocol.StatementResource;
+import com.facebook.presto.server.remotetask.HttpRemoteTaskFactory;
```

Comment:
It looks like it should go to a different commit

Reply:
@arhimondr : No. This happens in this commit -- it is because `HttpRemoteTaskFactory` class was moved to package `com.facebook.presto.server.remotetask`